qa/tasks/scripts/install_ceph_salt.sh

```diff
@@ -0,0 +1,21 @@
+set -ex
+
+REPO=$1
+BRANCH=$2
+if [ -n "$REPO" ]
+then
+	if [ ! -n "$BRANCH" ]
+	then
+		BRANCH="master"
+	fi
+	cd /root
+	git clone $REPO
+	cd ceph-salt
+	zypper -n install autoconf gcc python3-devel python3-pip python3-curses
```

Comment:
that can be sudo-ed command

Reply:
`scripts.run` by default runs the script as root so no need to sudo. See https://github.com/gekios/ceph/blob/c24a1f337d6d2b41a47f4a7e03d272a0e6452f53/qa/tasks/scripts.py#L21

Reply:
Interesting, while you test this script locally there is no sense to run 'zypper in' or 'zypper up' without sudo if you run it as regular user, from other side, what is the sense in a script which can only be executed as sudo, without any notice regarding that?

Reply:
When you test it locally, you need to know how it gets run in the CI, and run it in the same way.